docs/KNOWN_ISSUES.md

```diff
@@ -49,3 +63,31 @@ If you *must* call `player.reset()` then you'll have to manually restart the `vi
         // Add your plugin configuration here
     });
     ```
+
+## Setting start position on iOS safari
+
+On iOS Safari, there is a compatibility issue with the `loadedmetadata` event when attempting to set the start position
+of a video. This event, commonly used for this purpose on other browsers, may lead to unexpected behavior on iOS
+devices.
+
+### Issue Details
+
+- **Scenario:** Calling `player.on('loadedmetadata', () => player.currentTime(startPosition))`.
+- **Affected Browser:** Safari for iOS.
+- **Symptoms:** The video doesn't seek to the starting position and the seeking bar becomes stuck.
+
+### Workaround
+
+To address this issue on iOS Safari, it is recommended to use the `canplay` event instead of `loadedmetadata`.
+The `canplay` event is triggered when enough data has been loaded to start playback. While it might not be the preferred
```

Comment:
What about using `loadeddata` event instead?

Reply:
Sure ! That works too, I was afraid of this note in the [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/loadeddata_event) though : 

> Note: This event will not fire in mobile/tablet devices if data-saver is on in browser settings.

But after testing with the data sa ver on it seems to work just fine.